### PR TITLE
Make construction files and related code type-safe

### PR DESCRIPTION
**Pull request by jonkoops**

### CODE REVIEW

File: src/app/components/ImageViewer/ImageViewer.tsx [hunk 1]
```diff
@@ -159,7 +159,6 @@ const ImageViewer: React.FC<ImageViewerProps> = ({
             <ContextMenu
               handleDownload={handleDownload}
               downloadLoading={downloadLoading}
-              fileName={fileName}
```
Comment (royderks) [inline src/app/components/ImageViewer/ImageViewer.tsx:162]: Why is this prop removed? The users want to download the file with a the original filename

File: src/app/containers/ActualityContainer.jsx [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
-import LoadingIndicator from '../../shared/components/loading-indicator/LoadingIndicator'
-import { fetchWithoutToken } from '../../shared/services/api/api'
+import styled from 'styled-components'
 import environment from '../../environment'
+import { fetchWithoutToken } from '../../shared/services/api/api'
+import LoadingSpinner from '../components/LoadingSpinner/LoadingSpinner'
+
+const StyledLoadingSpinner = styled(LoadingSpinner)`
+  position: absolute;
+  top: 200px;
```
Comment (royderks) [inline src/app/containers/ActualityContainer.jsx:9]: Where does 200px come from?

File: src/app/components/ImageViewer/ImageViewer.tsx (hunk 1 above)
Comment (jonkoops) [reply src/app/components/ImageViewer/ImageViewer.tsx:162]: This was never working as the `download` attribute was set on a `button` element. Also the download is handled by a separate method passed in with a prop that does not support passing a file name.

File: src/app/containers/ActualityContainer.jsx (hunk 2 above)
Comment (jonkoops) [reply src/app/containers/ActualityContainer.jsx:9]: `<LoadingIndicator style={{ top: '200px' }} />` this original code

**Pull request merged**